analyzer/src/main/kotlin/managers/NPM.kt

```diff
@@ -270,7 +274,7 @@ open class NPM(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConf
 
             val module = Package(
                     id = Identifier(
-                            provider = NPM.toString(),
+                            provider = "NPM",
```

Comment:
I see why you cannot use `toString` anymore, but I don't like the hardcoded provider Strings. But I also have no good idea how to fix it, e.g. a `name` property in `PackageManager` would work here, but not when `Gradle` needs `"Maven"` for the package references.

Reply:
I had similar thoughts which is why I created the `pm-tostring` branch in parallel which has https://github.com/heremaps/oss-review-toolkit/commit/8e15035f233745ad2b5b29475c2919c58fef81ab. Let's discuss about that separately.